nested/db-postgres.json

```diff
@@ -60,7 +60,7 @@
                     "type": "firewallRules"
                 },
                 {
-                    "apiVersion": "2017-12-01",
+                    "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.